e2e-tests/tfm/cypress/e2e/pages/utilisation-reports/utilisationReportEditPaymentPage.js

```diff
@@ -3,6 +3,7 @@ const utilisationReportEditPaymentPage = {
   clickSaveChangesButton: () => cy.get('input[data-cy="save-changes-button"]').click(),
   clickDeletePaymentButton: () => cy.get('a[data-cy="delete-payment-button"]').click(),
   clickRemoveSelectedPaymentsButton: () => cy.get('input[data-cy="remove-selected-fees-button"]').click(),
+  clickBackLink: () => cy.get('[data-cy="back-link"]').click(),
```

Comment:
Can we move this into a different structure please - there is a PR open to have back link and other common buttons/links non-specific to the page:

https://github.com/UK-Export-Finance/dtfs2/pull/3453/files#diff-209e86a9ed4a341306fb6f673efa64efbf30abf821716d084696d7f69798084fR1

Happy to pair on this if you'd like, will only take 5mins 😎

Reply:
Changed to adopt this, thanks for the link